internal/pkg/utils/etcdhelper/assert.go

```diff
@@ -68,14 +81,26 @@ func AssertKeys(t assert.TestingT, client etcd.KV, expectedKeys []string, ops ..
 	return assert.Equal(t, expectedKeys, actualKeys)
 }
 
+// AssertKVsFromFile dumps all KVs from an etcd database and compares them with content of the file.
+// In the file, a wildcards can be used, see the wildcards package.
+func AssertKVsFromFile(t assert.TestingT, client etcd.KV, path string, ops ...AssertOption) bool {
```

Comment:
Added function to compare etcd state with a file, because `expected` state can have a lot of lines.